lib/facilities/ppms/client.rb

```diff
@@ -44,6 +44,7 @@ def pos_locator(params)
           providers = Facilities::PPMS::Response.from_provider_locator(response, request_params)
           providers.each do |provider|
             provider.posCodes = request_params[:posCodes]
+            provider.ProviderType = 'GroupPracticeOrAgency'
```

Comment:
I think a "why-style" comment is warranted here. It took me a good bit of time to realize that "GroupPracticeOrAgency" is a special string used in the provider serializer.